geometry_msgs/Twist  velocity_command

bool    motion_ackerman
bool    motion_pointsteer
bool    motion_translatory
bool    motion_skid
bool    motion_swerve

bool    motion_enable